### PR TITLE
Re-formatted the code base based on Prettier config

### PR DESCRIPTION
**Pull request by matchilling**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ A website for Equal Experts Germany
 ## Automated Responsive Regression testing with backstopJS
 
 I had some issues while developing the site with fixing css at one breakpoint only to discover later that the fix had broken a different breakpoint. Using BackstopJS, I developed a flow when developing new components doing automated Responsive regression testing, that goes :
-> Develope component with stubbed data -> Test -> Generate 'Good' Visual Reference -> Wire up component to real data -> Test -> Commit 
+
+> Develope component with stubbed data -> Test -> Generate 'Good' Visual Reference -> Wire up component to real data -> Test -> Commit
```
Comment (spacebaboon) [inline README.md:9]: 'Develop' or 'Developed'

File: gatsby-config.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-const languages = require('./languages');
+const languages = require("./languages")
```
Comment (spacebaboon) [inline gatsby-config.js:1]: single quotes are preferred these days, and are the default Prettier config, for example. Also, my preference is always terminating with a semicolon, but that's up to you :)

**Comment by spacebaboon** [comment]
This is pretty much all just converting single quotes to doubles, and and semicolon-terminated line endings. It's very hard to see where your meaningful changes are. 
Personally I prefer things as they were in both cases, but I'm not religious about it. But I would suggest codebase-wide code style changes like these should be in a separate PR, or at least a separate commit.

**Comment by matchilling** [comment]
Hi @spacebaboon 👋, sry for mixing things a little bit up. My [content related changes](https://github.com/EqualExperts/equalexperts-de/issues/5) got somehow lost as I wasn't familiar with the repo. @SimonKenyonShepard just granted me access to Contentful so that I can make these changes in the CMS.

**Comment by spacebaboon** [comment]
Ignore me, I'm an idiot! I got this confused with your PR for improving the German 😆 
... oh hang on, did you in fact just change this PR?

**Comment by matchilling** [comment]
> Ignore me, I'm an idiot! I got this confused with your PR for improving the German 😆

No worries, I actually did change the PR title after I noticed that my content changes were missing 😝

**Comment by spacebaboon** [comment]
damn, this is going to be a hard Friday, if I'm already this confused before 10am.

**Comment by matchilling** [comment]
> damn, this is going to be a hard Friday, if I'm already this confused before 10am.

Wow, it's Friday already ... there was a Wednesdayish feel to the air 🤔

**Comment by spacebaboon** [comment]
homer

**Comment by SimonKenyonShepard** [comment]
is it possible to add these stylechecks to the build?

**Comment by spacebaboon** [comment]
On my recent projects, we use a pre-commit hook via Husky to format the code, so it doesn't matter how each developer chooses to format their code, it will all get the same rules applied when being checked in.

**Comment by SimonKenyonShepard** [comment]
cool - sounds good - how do we do that?

**Comment by matchilling** [comment]
> On my recent projects, we use a pre-commit hook via Husky to format the code, so it doesn't matter how each developer chooses to format their code, it will all get the same rules applied when being checked in.

Good call, I'd suggest to [add the lint checker to the build pipeline](https://github.com/EqualExperts/equalexperts-de/pull/6) anyway to ensure a consistent format.

**Pull request merged**